src/app/components/display-hash/display-hash.component.ts

```diff
@@ -0,0 +1,53 @@
+import { DatasetInfo } from "./../../interface/navigation.interface";
+import { NavigationService } from "./../../services/navigation.service";
+import { Component, Input } from "@angular/core";
+import { Clipboard } from "@angular/cdk/clipboard";
+
+@Component({
+    selector: "app-display-hash",
+    templateUrl: "./display-hash.component.html",
+    styleUrls: ["./display-hash.component.sass"],
+})
+export class DisplayHashComponent {
+    @Input() public value: string;
+    @Input() public datasetInfo: DatasetInfo;
+    @Input() public showCopyButton?: boolean = false;
+    @Input() public class?: string;
+    @Input() public isNavigable?: boolean = true;
```

Comment:
? not necessary, default value is enough

Reply:
Fixed